script.js

```diff
@@ -55,61 +57,77 @@ stories.innerHTML += `<div class="recent_work_div flex">
     name: 'Website Portfolio',
     desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has beentheindustry's standard",
     featuredImg: '',
-    technologies: ['html', 'bootstrap', 'ruby'],
+    technologies: ['html', 'Bootstrap', 'Ruby'],
     liveLink: '#',
-    sourceLink: '#',
+    sourceLink: 'https://github.com/3nityGitHub/MOBILE_FIRST-PROJECT',
   },
   {
     id: '_4',
+    heading: 'Multi Post Stories',
     name: 'Profesional Art Printing Data',
     desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has beentheindustry's standard",
     featuredImg: '',
-    technologies: ['html', 'bootstrap', 'ruby'],
+    technologies: ['html', 'Bootstrap', 'Ruby'],
     liveLink: '#',
-    sourceLink: '#',
+    sourceLink: 'https://github.com/3nityGitHub/MOBILE_FIRST-PROJECT',
   },
   {
     id: '_5',
+    heading: 'Multi Post Stories',
     name: 'Data Dashboard Healthcare',
     desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has beentheindustry's standard",
     featuredImg: '',
-    technologies: ['html', 'bootstrap', 'ruby'],
+    technologies: ['html', 'Bootstrap', 'Ruby'],
     liveLink: '#',
-    sourceLink: '#',
+    sourceLink: 'https://github.com/3nityGitHub/MOBILE_FIRST-PROJECT',
   },
   {
     id: '_6',
+    heading: 'Multi Post Stories',
     name: 'Website Portfolio',
-    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has beentheindustry's standard",
+    desc: "A daily selection of privately personalized reads; no accounts or sign-ups required. has beentheindustry's standard ",
     featuredImg: '',
-    technologies: ['html', 'bootstrap', 'ruby'],
+    technologies: ['html', 'Bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: 'https://github.com/3nityGitHub/MOBILE_FIRST-PROJECT',
+  },
+  {
+    id: '_7',
+    heading: 'Multi Post Stories',
+    name: 'Keeping track of hundreds  of components website',
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.Lorem Ipsum is simply dummy text of the printing and typesetting industry. <br> <br>Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, A daily selection of privately personalized reads; no accounts or sign-ups required.",
+    featuredImg: 'image folder/popup image.png',
+    technologies: ['html', 'Bootstrap', 'Ruby on rails'],
     liveLink: '#',
-    sourceLink: '#',
+    sourceLink: 'https://github.com/3nityGitHub/MOBILE_FIRST-PROJECT',
   },
 ];
 
-for (let i = 0; i < projectsArr.length; i += 1) {
+for (let i = 0; i <= 5; i += 1) {
   const workCard = projectsArr[i];
   const projects = document.getElementById('project_cards');
   projects.innerHTML += `<div class="project_background">
```

Comment:
- [ ] I am sorry for this unexpected bug but at the moment, your `project cards` are not showing as you intend that they do which makes implementing the popup (requirement) even more difficult. Here is a [video](https://watch.screencastify.com/v/0UIpoq4dcrsWf186iJNd) that explains my findings on your page, and maybe a screenshot will give you a clue too:
![image](https://user-images.githubusercontent.com/99715640/233415787-0eabb533-0726-49c9-84aa-565bef39b14e.png)
- [ ] Kindly check this feature for possible anomalies :pray: and resolve with your expertise.

Reply:
@AbayomiOlaoye Thank you for your review, please are you available for a quick call on zoom?

Reply:
@3nityGitHub , I'm certain that I will be in exactly an hour time. Is that possible? Let me know if you still need me then. I'm sorry. I have an assignment immediately after the stand-up session now.